src/library.js

```diff
@@ -1625,7 +1625,9 @@ LibraryManager.library = {
       if (flag & 256) { // RTLD_GLOBAL
         for (var ident in lib_module) {
           if (lib_module.hasOwnProperty(ident)) {
-            Module[ident] = lib_module[ident];
+            if (ident.substr(0,1) == '_') {
```

Comment:
Your recognition is correct. ( This just copies over compiled methods (which start with _).)
I hope the symbols are located in not Module._* but Module.symbols[*].
I wonder changing symbol location(Module.symobls[*]) affects performance.